conf/schemas/cloudtrail.json

```diff
@@ -22,26 +22,31 @@
   "cloudtrail:events": {
     "schema": {
       "additionalEventData": {},
+      "addendum": {},
       "apiVersion": "string",
       "awsRegion": "string",
       "errorCode": "string",
       "errorMessage": "string",
+      "eventCategory": "string",
       "eventID": "string",
       "eventName": "string",
       "eventSource": "string",
       "eventTime": "string",
       "eventType": "string",
       "eventVersion": "string",
+      "insightDetails": {},
       "managementEvent": "boolean",
       "readOnly": "boolean",
       "recipientAccountId": "string",
       "requestID": "string",
       "requestParameters": {},
       "resources": [],
       "responseElements": {},
+      "sessionCredentialFromConsole": "boolean",
       "serviceEventDetails": {},
       "sharedEventID": "string",
       "sourceIPAddress": "string",
+      "tlsDetails": {},
```

Comment:
```
tlsDetails
Shows information about the Transport Layer Security (TLS) version, cipher suites, and the FQDN of the client-provided host name of a service API call. Contents include the following. CloudTrail still logs partial TLS details if expected information is missing or empty. For example, if the TLS version and cipher suite are present, but the HOST header is empty, available TLS details are still logged in the CloudTrail event.

If sessionCredentialFromConsole is present with a value of true, tlsDetails is present in an event record only if an external client was used to make the API call.

tlsVersion - The TLS version of a request.

cipherSuite - The cipher suite (combination of security algorithms used) of a request.

clientProvidedHostHeader - The FQDN of the client that made the request.

Since: 1.08

Optional: True

```